core/class/descendants_spec.rb

```diff
@@ -1,7 +1,7 @@
 require_relative '../../spec_helper'
 require_relative '../module/fixtures/classes'
 
-ruby_version_is '3.1' do
+ruby_version_is '3.2' do
```

Comment:
I suspended these tests since Ruby core team decided not to have Ruby 3.1.0 include `Class#descendants`.

CI failed due to missing the method: https://github.com/ruby/spec/runs/4638548211?check_suite_focus=true

```
Class#descendants has 1 entry per module or class ERROR
NoMethodError: undefined method `descendants' for ModuleSpecs::Parent:Class

      ModuleSpecs::Parent.descendants.should == ModuleSpecs::Parent.descendants.uniq
                         ^^^^^^^^^^^^
```

Refs:
- https://github.com/ruby/ruby/pull/5309
- https://bugs.ruby-lang.org/issues/14394#note-38